Graphics/ShaderTools/CMakeLists.txt

```diff
@@ -43,7 +43,11 @@ if((PLATFORM_WIN32 AND NOT MINGW_BUILD) OR PLATFORM_UNIVERSAL_WINDOWS OR PLATFOR
     set(DXC_SUPPORTED TRUE)
 endif()
 
-if (DXC_SUPPORTED)
+if(NOT DILIGENT_NO_DIRECT3D11 OR NOT DILIGENT_NO_DIRECT3D12)
+  set(DXC_NEEDED TRUE)
+endif()
```

Comment:
This logic is incorrect. On Linux, for example, there is no D3D11 and D3D12 even when neither DILIGENT_NO_DIRECT3D11 nor DILIGENT_NO_DIRECT3D12 is defined, but DXC is still needed.
The correct check is if (D3D11_SUPPORTED or D3D12_SUPPORTED)